fuel-core-interfaces/src/model/messages.rs

```diff
@@ -1,12 +1,11 @@
+use fuel_vm::prelude::Input;
```

Comment:
Could you use `crate::common::fuel_vm::prelude::Input`? It will collapse everything into one import